src/instructlab/configuration.py

```diff
@@ -684,6 +684,313 @@ class Config(BaseModel):
     )
 
 
+EIGHT_GPU_TRAIN_AH = _train(
```

Comment:
Lot of these parameters are repeated among different configurations. I would recommend making them defaults, or derive a new class from train.

Reply:
Internally its super helpful for us to have separate train profiles, eventually we want to derive a new class but that is out of scope for this PR. These all existed, I just moved them around